one-ui/ui/src/components/entities/matching/create-edit-matching-dialog/create-edit-matching-dialog.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CreateEditMatchingDialog from './create-edit-matching-dialog';
+
+describe('Create/Edit Matching artifact component', () => {
+  let wrapper: any;
+
+  beforeEach(() => {
+    wrapper = shallow(<CreateEditMatchingDialog data/>)
+  });
+
+  test('Create/Edit Matching Dialog renders ', () => {
+    expect(wrapper.find('#name').length).toEqual(1)
+    expect(wrapper.find('#description').length).toEqual(1)
+    expect(wrapper.find('#srcType').length).toEqual(1)
+    expect(wrapper.find('#srcQuery').length).toEqual(1)
```

Comment:
this should equal 0. Collection radio box is checked by default so srcQuery input div will be hidden